### ISSUE TITLE
[INFO] TypeError: super() takes at least 1 argument (0 given)

### ISSUE THREAD

**Issue by CrimsonGlory** [john-kurkowski/tldextract#201]
Just because some people will obviously get this stracktrace I'll post this info. A few hours ago an update of requests-file from 1.4.3 to 1.5.0 broke tldextract on python:2.7. The fix is to harcode request-file==1.4.3 (or update to python 3 obviously).

Python 3 works:
```
# docker run --rm -ti python:3 bash -c "pip --quiet install tldextract && pip freeze && python -c \"import tldextract;print(tldextra
ct.extract('www.google.com'))\""
certifi==2020.4.5.1
chardet==3.0.4
idna==2.9
requests==2.23.0
requests-file==1.5.0
six==1.14.0
tldextract==2.2.2
urllib3==1.25.9
ExtractResult(subdomain='www', domain='google', suffix='com')
```

Python 2.7 with requests-file==1.5.0 doesn't:
```
# docker run --rm -ti python:2.7 bash -c "pip --quiet install tldextract && pip freeze && python -c \"import tldextract;print(tldext
ract.extract('www.google.com'))\""
on of pip will drop support for Python 2.7.
certifi==2020.4.5.1
chardet==3.0.4
idna==2.9
requests==2.23.0
requests-file==1.5.0
six==1.14.0
tldextract==2.2.2
urllib3==1.25.9
virtualenv==16.6.0
Traceback (most recent call last):
  File "<string>", line 1, in <module> 
  File "/usr/local/lib/python2.7/site-packages/tldextract/tldextract.py", line 353, in extract
    return TLD_EXTRACTOR(url)
  File "/usr/local/lib/python2.7/site-packages/tldextract/tldextract.py", line 233, in __call__
    suffix_index = self._get_tld_extractor().suffix_index(translations)
  File "/usr/local/lib/python2.7/site-packages/tldextract/tldextract.py", line 276, in _get_tld_extractor
    self.cache_fetch_timeout
  File "/usr/local/lib/python2.7/site-packages/tldextract/remote.py", line 33, in find_first_response
    session.mount('file://', FileAdapter())
  File "/usr/local/lib/python2.7/site-packages/requests_file.py", line 14, in __init__
    super().__init__()
TypeError: super() takes at least 1 argument (0 given)
```

python 2.7 with requests-file==1.4.3 does:
```
# docker run --rm -ti python:2.7 bash -c "pip --quiet install tldextract requests-file==1.4.3 && pip freeze && python -c \"import tldextract;print(tldextract.extract('www.google.com'))\""
certifi==2020.4.5.1
chardet==3.0.4
idna==2.9
requests==2.23.0
requests-file==1.4.3
six==1.14.0
tldextract==2.2.2
urllib3==1.25.9
virtualenv==16.6.0
ExtractResult(subdomain='www', domain='google', suffix='com')
```

**Comment by singingwolfboy** [comment]
This should now be fixed with requests-file version 1.5.1.

**Comment by john-kurkowski** [comment]
Fixed upstream. Thanks for pushing that through.
